Sources/IO/Geometry/STLReader/index.js

```diff
@@ -77,6 +96,12 @@ function vtkSTLReader(publicAPI, model) {
     model.parseData = content;
 
     // Binary parsing
+    // Header
+    const headerData = new Uint8Array(content, 0, 80);
+    const headerStr = BinaryHelper.arrayBufferToString(headerData);
```

Comment:
@jourdain this works, but should I initialize `headerData` as something else than a `Uint8Array`?

Reply:
Check with @floryst

Reply:
`arrayBufferToString` should take in an ArrayBuffer, but it'll work with UInt8Arrays.

Note the fallback code in `arrayBufferToString` will make a copy of your input if it's UInt8Array.

Reply:
used `slice()` to provide ArrayBuffer of proper offset and length.